filament/src/materials/dofCombine.mat

```diff
@@ -0,0 +1,72 @@
+material {
+    name : DepthOfFieldCombine,
+    parameters : [
+        {
+            type : sampler2d,
+            name : color,
+            precision: medium
+        },
+        {
+            type : sampler2d,
+            name : dof,
+            precision: medium
+        },
+        {
+            type : sampler2d,
+            name : tiles,
+            precision: medium
+        },
+        {
+            type : sampler2d,
+            name : alpha,
+            precision: medium
+        }
+    ],
+    variables : [
+        vertex
+    ],
+    domain : postprocess,
+    depthWrite : false,
+    depthCulling : false
+}
+
+vertex {
+    void postProcessVertex(inout PostProcessVertexInputs postProcess) {
+    postProcess.vertex.xy = postProcess.normalizedUV;
+}
+}
+
+fragment {
+
+#include "dofUtils.fs"
+
+void dummy(){}
```

Comment:
remove?

Reply:
ah yes. I have a bunch of those, that's to squash CLion syntax highlighting, it makes it hard to work on the code without it :-\